website/content/docs/audit/syslog.mdx

```diff
@@ -43,3 +43,18 @@ these device-specific options:
 - `facility` `(string: "AUTH")` - The syslog facility to use.
 
 - `tag` `(string: "vault")` - The syslog tag to use.
+
+
+## Notes
```

Comment:
```suggestion
## Entry size causing syslog audit device errors
```
I think it would be good if this section's name was more specific, since it's visible on the navbar on the right.